BitsAI updates (#22336)
DataDog/documentation: content/en/bits_ai/managing_incidents.md

---
title: Managing Incidents
kind: guide
further_reading:
- link: "bits_ai/"
  tag: "Documentation"
  text: "Bits AI Overview"
- link: "bits_ai/getting_started"
  tag: "Documentation"
  text: "Getting Started"
- link: "bits_ai/query_examples"
  tag: "Documentation"
  text: "Example Natural Language Queries"
---

## Overview

<div class="alert alert-warning">The following features are part of the <a href="https://www.datadoghq.com/product/incident-management/">Datadog Incident Management</a> product.</div>

Bits AI simplifies incident management processes, improves collaboration, and provides valuable support for incident responders, making it a beneficial tool for efficient incident resolution.

## Prerequisites

- Your Datadog account must be connected to Slack. Running the `/dd` connect command automatically initiates this process, which can be completed by following the prompts.

> - In **[Incident > Settings > Integrations][3] > Slack**, enable the **"Push Slack channel messages to the incident timeline"** and **"Activate Bits AI features in incident Slack channels for your organization"** toggles. This allows Datadog to ingest Slack conversations into the incident timeline to generate summaries and postmortems. **Note**: a Slack workspace can only have Bits AI activated for one Datadog Organization.   

Reviewer: ```suggestion
- In **[Incident > Settings > Integrations][3] > Slack**, enable the **Push Slack channel messages to the incident timeline** and **Activate Bits AI features in incident Slack channels for your organization** toggles. This allows Datadog to ingest Slack conversations into the incident timeline to generate summaries and postmortems. **Note**: a Slack workspace can only have Bits AI activated for one Datadog organization.   
```

seems the quotes are not necessary

- Incident channels you'd like to work with Bits AI must be prefixed with `#incident-`. 
- To ask Bits AI questions about incidents from any Slack channel, you must invite Bits AI to that channel. Run the `@Datadog` command and follow the instructions on the screen. 

{{< img src="bits_ai/managing_incidents/bitsai_slack_prerequisites.png" alt="Slack integration settings in Datadog" style="width:90%;">}}

## View incident summaries

When you join an incident channel in Slack, you automatically receive a summary of the incident. The channel must be connected to Incident Management and **have at least ten messages**. The summary, which is only visible to you, does not persist across reloads, between desktop and mobile apps, or across sessions.

> At any time, you can ask for a fresh summary by asking `@DataDog Give me a summary of this incident`. 

Reviewer: ```suggestion
At any time, you can ask for a fresh summary by asking `@Datadog Give me a summary of this incident`. 
```

seems like it should not be camelcased

## Search across your entire incident history and ask questions

> You can ask Bits to find incidents that you're looking for. For instance,

Reviewer: ```suggestion
You can ask Bits AI to find incidents that you're looking for. For instance,
```

- `@Datadog How many incidents are currently ongoing?`
- `@Datadog Show me all Sev-1 incidents that occurred in the past week`

You can then investigate further and ask questions about those incidents, like `@Datadog What was the root cause of incident-123?` or `@Datadog What remediation actions did the responders take in incident-123?`

> Bits AI can also perform semantic search for related incidents. If you're responding to an incident, you can ask Bits AI to find other active incidents that look similar to the one you're in (`@Datadog Are there any related incidents?`). Bits AI looks for incidents that were active within the past 2 hours. You can also specify the time frame you want Bits to look across. If you say `@Datadog Find me incidents related to DDOS attacks from the past month`, Bits AI returns both active and resolved DDOS incidents from the past month. 

Reviewer: ```suggestion
Bits AI can also perform semantic searches for related incidents. If you're responding to an incident, you can ask Bits AI to find other active incidents that look similar to the one you're in (`@Datadog Are there any related incidents?`). Bits AI looks for incidents that were active within the past two hours. You can also specify the time frame you want Bits AI to look across. If you say `@Datadog Find me incidents related to DDOS attacks from the past month`, Bits AI returns both active and resolved DDOS incidents from the past month. 
```

> Or if you suspect there's an issue before an incident is even declared, you can ask Bits AI a question like `@Datadog A customer is unable to check out. Is there an incident?` or `@Datadog Are there any incidents now impacting the payments service?` 

Reviewer: ```suggestion
Or, if you suspect there's an issue before an incident is even declared, you can ask Bits AI a question like `@Datadog A customer is unable to check out. Is there an incident?` or `@Datadog Are there any incidents now impacting the payments service?` 
```

Bits AI simplifies incident management processes, improves collaboration, and provides valuable support for incident responders, making it a beneficial tool for efficient incident resolution.

## Prerequisites

- Your Datadog account must be connected to Slack. Running the `/dd` connect command automatically initiates this process, which can be completed by following the prompts.
- In **[Incident > Settings > Integrations][3] > Slack**, enable the **"Push Slack channel messages to the incident timeline"** and **"Activate Bits AI features in incident Slack channels for your organization"** toggles. This allows Datadog to ingest Slack conversations into the incident timeline to generate summaries and postmortems. **Note**: a Slack workspace can only have Bits AI activated for one Datadog Organization.   
- Incident channels you'd like to work with Bits AI must be prefixed with `#incident-`. 
- To ask Bits AI questions about incidents from any Slack channel, you must invite Bits AI to that channel. Run the `@Datadog` command and follow the instructions on the screen. 

> {{< img src="bits_ai/managing_incidents/integration-settings.png" alt="Slack integration settings in Datadog" style="width:90%;">}}

Reviewer 2: This screenshot is outdated, will need to grab a new one

## View incident summaries

> When you join an incident channel in Slack, you automatically receive a summary of the incident. The channel must be connected to Incident Management and have at least ten messages. The summary, which is only visible to you, does not persist across reloads, between desktop and mobile apps, or across sessions.

Reviewer 2: Can we bold the part that says "at least ten messages"? It's a common issue that people run into. They try summarizing when there are only 1 or 2 messages and it fails because we can't generate a summary with that little content.

Author: ```suggestion
When you join an incident channel in Slack, you automatically receive a summary of the incident. The channel must be connected to Incident Management and **have at least ten messages**. The summary, which is only visible to you, does not persist across reloads, between desktop and mobile apps, or across sessions.
```

At any time, you can ask for a fresh summary by asking `@DataDog Give me a summary of this incident`. 

## Search across your entire incident history and ask questions

You can ask Bits to find incidents that you're looking for. For instance,
- `@Datadog How many incidents are currently ongoing?`

> - `@Datadog A customer is unable to check out. Is there an incident?`

Reviewer 2: ```suggestion
- `@Datadog A customer is unable to check out. Is there an incident?`
```

Author: ```suggestion
```

- `@Datadog Show me all Sev-1 incidents that occurred in the past week`

> - `@Datadog Were there any incidents in the past month that impacted the payments service?` 

Reviewer 2: ```suggestion
- `@Datadog Were there any incidents in the past month that impacted the payments service?` 
```

Reviewer 2: Already using these examples in the following paragraphs

Author: Will remove
```suggestion
```
